### DISCUSSION TITLE
DeviceTwinStatusCallBack method is not available from the version 2.2.0 of iot-device-client

### DISCUSSION THREAD

**Discussion by soloxio** [Azure/azure-iot-sdk-java#1767] [Q&A]
Hello everyone,

I am facing an issue with the DeviceTwinStatusCallBack method after upgrading the library iot-device-client to verion 2.2.0.

In the past, when I used the version 1.x of iot-device-client, there was a method DeviceTwinStatusCallBack , but now it's no any such kind of this method in version 2.2.0.

Who can help me the same method to use?

Thanks in advance!

**Comment by soloxio** [comment]
Is there any one that can help me?

**Reply by abhipsaMisra** [reply]
Is this the right repository (.NET SDK) for the code that you're referencing?

I ask because:
- The publicly released 1.x.x versions of the .NET SDK don't have any method or property called `DeviceTwinStatusCallBack`.
- Our nuget package is called `Microsoft.Azure.Devices.Client`, and not `iot-device-client`.
- We haven't released a 2.2.x version yet. Our v2 version is still under a `preview` tag and is versioned as `2.0.0-preview00x`.

**Reply by soloxio** [reply]
Thanks @abhipsaMisra for your feedback. I mean this:
https://github.com/Azure/azure-iot-sdk-java/releases

**Reply by abhipsaMisra** [reply]
You'll need to open an issue on the linked Java repository's page and then one of the maintainers will be able to help you with that: https://github.com/Azure/azure-iot-sdk-java/issues
This repository is for the Azure IoT .NET SDK code.

**Reply by abhipsaMisra** [reply]
Transferred the issue to the Java repository.

**Reply by soloxio** [reply]
Thanks a lot !
